generator/config.js

```diff
@@ -0,0 +1,60 @@
+const { existsSync } = require("fs");
+const { resolve } = require("path");
+
+const defaultConfig = {
+  excludes: [],
+  force: false,
+  format: "js",
+  input: "graphql-schema.json",
+  modelsOnly: false,
+  outDir: "src/models",
+  roots: [],
+  noReact: false,
+}
+
+exports.getConfig = function getConfig(configPath) {
+  let configLocation
+  if (configPath) {
+    configLocation = resolve(process.cwd(), configPath)
```

Comment:
Looking through this another, I noticed that relying on `process.cwd` might be tricky, because, if the script is initiated from one of the project's sub directories, it might not find it (afaik an package.json script doesn't automatically have the project root as cwd). But I think we can directly use this package to find the correct file: https://github.com/sindresorhus/find-up

Reply:
Wouldn't be better to go with https://github.com/davidtheclark/cosmiconfig right away which takes care of such edge cases? Feels weird to be reinventing a wheel once again.